### PR TITLE
[ALLUXIO-1775] Add unit tests for alluxio.wire.ThriftUtils

### PR DESCRIPTION
**Pull request by schernolyas**
https://alluxio.atlassian.net/browse/ALLUXIO-1775

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,4 +40,6 @@
 dependency-reduced-pom.xml
 target/
 tests.log*
-**/nb-configuration*.xml
\ No newline at end of file
+**/nb-configuration*.xml
+/underfs/s3/nbproject/
+/underfs/oss/nbproject/
```
Comment (aaudiber) [inline .gitignore]: should we ignore `**/nbproject/`?

Comment (schernolyas) [inline .gitignore]: Hi @aaudiber !
I am using Netbeans :-) and ... I added Netbeans-specific files to ignore list

Comment (aaudiber) [inline .gitignore]: if we ignore `**/nbproject/` instead, it should ignore those two nbproject directories as well as any other nbproject directories that might be needed in other projects. The `**` matches any number of directories.

**Comment by schernolyas** [comment]
Hi @aaudiber !

Please recheck!

**Pull request merged**